src/main/java/com/maxmind/geoip2/model/IspResponse.java

```diff
@@ -7,20 +7,22 @@
  */
 public class IspResponse extends AbstractResponse {
 
-    @JsonProperty("autonomous_system_number")
-    private Integer autonomousSystemNumber;
+    private final Integer autonomousSystemNumber;
+    private final String autonomousSystemOrganization;
+    private final String isp;
+    private final String organization;
+    private final String ipAddress;
 
-    @JsonProperty("autonomous_system_organization")
```

Comment:
I suspect the JSON serialization for these changes due to your PR. I guess we should add a test for the serialization of all of these as some people do serialize these models to JSON with Jackson.